strings/wanted_message.json

```
@@ -0,0 +1,110 @@
+{
+	"basemessage": [
+		"Fugitive from the law due to",
+		"Needs to be interrogated for information about",
+		"Wanted by The Syndicate for",
+		"Ransomable to Nanotrasen for",
+		"Has exploitable information about"
+	],
+	"verb": [
+		"murdering",
+		"killing",
+		"accidentally destroying",
+		"destroying",
+		"knowing information about",
+		"stealing",
+		"slipping",
+		"sabotaging",
+		"robusting",
+		"collaborating with",
+		"being close friends with",
+		"cloning",
+		"befriending",
+		"bombing",
+		"kidnapping",
+		"pretending to be",
+		"seducing",
+		"ignoring",
+		"assassinating",
+		"drinking",
+		"hating",
+		"slapping",
+		"swatting",
+		"skreeing at",
+		"toolboxing",
+		"swirling",
+		"wearing",
+		"[REDACTED]"
+	],
+	"noun": {
+		"secret plans": 50,
+		"the hand teleporter": 50,
+		"an NT CentCom uniform": 50,
+		"a supermatter shard": 50,
+		"internal Syndicate documents": 50,
+		"experimental Nanotrasen technology": 50,
+		"bluespace crystals": 50,
+		"a cult": 50,
+		"shapeshifting creatures": 50,
+		"a bluespace artillery": 50,
+		"a DNA vault": 50,
+		"a toolbox": 10,
+		"a bar of soap": 10,
+		"Ian": 10,
+		"a Diona": 10,
+		"a Drask": 10,
+		"a Golem": 10,
+		"a Grey": 10,
+		"a Human": 10,
+		"a Kidan": 10,
+		"a Monkey": 10,
+		"a Nucleation": 10,
+		"a Plasmaman": 10,
+		"a Shadow": 10,
+		"a Skeleton": 10,
+		"a Skrell": 10,
+		"a Slime": 10,
+		"a Slime Person": 10,
+		"a Tajaran": 10,
+		"a Vox": 10,
+		"a Vulpkanin": 10,
+		"a Wryn": 10,
+		"an IPC": 10,
+		"an Unathi": 10,
+		"[REDACTED]": 10,
+		"two million credits": 10,
+		"one billion credits": 5,
+		"research on floor clowns": 5,
+		"Officer Beepsky": 5,
+		"clown tears": 5,
+		"John F Kennedy 2": 5,
+		"Nanotrasen's swimsuit calendar": 5,
+		"a suspicious bus": 5,
+		"administrators": 5,
+		"cat ears": 5,
+		"the anime archive": 5,
+		"the CMO's cat": 5,
+		"WetSkrell.nt": 5,
+		"YALPER": 1,
+		"absolutely nothing": 1,
+		"Cuban Pete": 1,
+		"your mother": 1,
+		"four cups of coffee a day": 1,
+		"ducks": 1,
+		"NanoUI": 1
+	},
+	"location": {
+		"on Space Station 13": 50,
```

```suggestion
		"on NT-SS13": 50,
```
I would say cyberiad but other maps have different names

---

You can use other station names since it's not relevant where it happened as it's just flavor